### PR TITLE
Fix and simplify upload-button

### LINKED ISSUE #770: Image Upload button not working
The image upload button stopped working for replies, comments, issues, etc.

I believe the culprit is that it is not a `button` but a `label` element. Seems like this is a mistake and that a click handler is likely missing/not working because of it.

Here is an image of the button I'm referring to: 
screen shot 2017-10-26 at 9 34 44 am

And here is an image of the markup I'm seeing when I inspect it: 
screen shot 2017-10-26 at 9 37 54 am

### PR DESCRIPTION
**Pull request by bfred-it**
Fixes #770

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
It doesn't work when a click a line in a PR diff to create an inline comment and then the upload button there.

File: extension/content.css [hunk 1]
```diff
@@ -95,21 +95,13 @@
 .toolbar-group:last-child {
 	display: inline-block;
 }
```
Comment (bfred-it) [inline extension/content.css]: test

**Comment by bfred-it** [comment]
Ok. Now it seems to work now. Sorry for the many pushes.

**Comment by sindresorhus** [comment]
Works fine now :)

**Pull request merged**